portfolio/src/main/webapp/script.js

```diff
@@ -39,33 +28,114 @@ window.onload = loadComments();
-        singleComment.innerText = comments[i].comment;
-        allCommentsList.append(singleComment);
+        allCommentsList.append(createSingleComment[i]);
       }
     });
 }
 
+function createSingleComment(comment){
+    const commentDiv = document.createElement('div');
+
+    const nameTitle = document.createElement('h3');
+    nameTitle.innerHTML = "Name: " + sanitizeHTML(comment.name);
+    commentDiv.appendChild(nameTitle);
+
+    const commentContent = document.createElement('p');
+    const moodReceived = comment.mood;
+    if (moodReceived === "happy"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😊 ";
+    } else if(moodReceived === "heart"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😍 ";
+    } else if(moodReceived === "surprised"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😯 ";
+    } else if(moodReceived === "sad"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😥 ";
+    } else {
+            commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment);
+    }
+    commentDiv.appendChild(commentContent);
+
+    const deleteCommentBtn = document.createElement("button");
+    deleteCommentBtn.innerHTML("<p> delete </p>");
+    deleteCommentBtn.addEventListener('click', () => {
+        deleteSingleComment(comment);
+    });
+    commentDiv.appendChild(deleteCommentBtn);
+
+    return commentDiv;
+}
+
+//Prevents HTML injection by removing markup from code
+function sanitizeHTML(str) {
+    const temp = document.createElement('div');
+    temp.textContent = str;
+    return temp.innerHTML;
+}
+
 //Performs POST request to /delete-data and fetches data again so comments are deleted
 function deleteComments(){
     fetch('delete-data', {method: 'POST'})
         .then(loadComments());
 }
 
+//Performs POST request to /delete-single-comment and fetches remaining comments 
+function deleteSingleComment(comment){
+    const id = comment.id;
+     fetch('delete-single-comment?id=' + id, {method: 'POST'})
+        .then(loadComments());
+}
+
 //Returns choosen value of comments
 function commentAmount(){
-    var amount = document.getElementById("number");
-    var value = amount.value
+    const amount = document.getElementById("number");
+    const value = amount.value
     return value
 }
 
+let currentPage = 0;
+const pages;
+const numberOfPages;
+
+//Disables button based on current page 
+function diableButton(){
+    const nextButton = document.getElementById('btnNext');
+    const prevButton = document.getElementById('btnPrev');
+    nextButton.disabled=false;
+    prevButton.disabled=false;
+    if (currentPage === 0) {
+        btnNext.disabled=false;
+        btnPrev.disabled=true;
+    } else if (currentPage === numberOfPages - 1) {
+        btnNext.disabled=true;
+        btnPrev.disabled=false;
+  }
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation

Block indentation SHOULD be 2 spaces, not 4, not tabs, but I'm not going to enforce something like that as long as your self-consistent (2 spaces, or 4 spaces, or tabs everywhere).

The problem here is this curly brace doesn't line up with anything in the above method, so pick a style for this whole file and make sure things line up.